### PR TITLE
Changes for the Updated fields in the XSD for the November release

### PR DESCRIPTION
**Pull request by sharathkatteri**

### CODE REVIEW

File: lib/net/authorize/api/contract/v1/PaymentScheduleType.php [hunk 1]
```diff
@@ -73,8 +73,7 @@ public function getStartDate()
      */
     public function setStartDate(\DateTime $startDate)
     {
-        $strDateOnly = $startDate->format('Y-m-d');
-        $this->startDate = \DateTime::createFromFormat('!Y-m-d', $strDateOnly); 
+        $this->startDate = $startDate;
```
Comment (gnongsie) [inline lib/net/authorize/api/contract/v1/PaymentScheduleType.php]: Revert this change.

Comment (sharathkatteri) [reply lib/net/authorize/api/contract/v1/PaymentScheduleType.php]: Reverted.
